packages/rancher-vsphere-cpi/charts/templates/_helpers.tpl

```diff
@@ -5,3 +5,14 @@
 {{- "" -}}
 {{- end -}}
 {{- end -}}
+
+{{- /* Return the image's repository and tag corresponding to the cluster's kubernetes version. */ -}}
+{{- define "get_image" -}}
+{{- $kubeVersion := printf "%s.%s" .Capabilities.KubeVersion.Major .Capabilities.KubeVersion.Minor -}}
+{{- if hasKey .Values.cloudControllerManager.image $kubeVersion -}}
+{{- $imageForKubeVersion := get .Values.cloudControllerManager.image $kubeVersion -}}
+{{- printf "%s:%s" $imageForKubeVersion.repository $imageForKubeVersion.tag -}}
+{{- else -}}
+{{- required (printf "unsupported Kubernetes version: %s" $kubeVersion) "" -}}
```

Comment:
Can we add which versions are currently supported to this error message?

Reply:
Adding it in next commit